watchApp/watchApp.xcodeproj/project.pbxproj

```diff
@@ -11,7 +11,7 @@
 		9B22A659220A6B3300A871AA /* PatientViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9B22A658220A6B3300A871AA /* PatientViewController.swift */; };
```

Comment:
Don't push project.pbxproj -- only push files that you directly worked on like VC's and storyboard